File: lksh2017/Day1/d1_4.cpp
#include <iostream>
#include <vector>
#include <fstream>
#include <string>
#include <algorithm>
/**
Okey
*/

using namespace std;

bool cmp(string a, string b){
    return a + b > b + a;
}

int main() {
    ofstream cout("number.out");
    ifstream cin("number.in");
    string a;
    vector<string> Arr;
    string ans;
    int n = 0;
    while (cin >> a) {
        Arr.push_back(a);
        n++;
    }

    sort(Arr.begin(), Arr.end(), cmp);

    for (auto i : Arr) {
        ans += i;
    }
    cout << ans;
}

File: lksh2017/Day1/d1_4.py
print('21'>'2')
def srt(a, b):
    return True
fin = open("number.in", "r")
fout = open("number.out", "w")
Arr = fin.readlines()
for i in range(len(Arr)):
    Arr[i] = list(Arr[i])
    if Arr[i][-1] == '\n':
        Arr[i].pop(-1)
Arr.sort(reverse = True)
Ans = []
for i in Arr:
    Ans += i
str = ''.join(Ans)
fout.write(str)
